spec/std/json/serializable_spec.cr

```diff
@@ -1122,6 +1116,6 @@ describe "JSON mapping" do
 
   it "fixes #13337" do
     JSONSomething.from_json(%({"value":{}})).value.should_not be_nil
-    JSONSomethingElse.from_json(%({"value":{}})).value.should_not be_nil
+    JSONAttrValue(JSONSomething).from_json(%({"value":{}})).value.should_not be_nil
```

Comment:
So I checked again, and this line is now completely redundant; there is no longer a dependency on call order and `JSONSomething.from_json` alone reproduces the original error